frontend/packages/component-library/src/cms/iframe/index.ts

```diff
@@ -0,0 +1,2 @@
+export type {IframeProps} from './Iframe';
```

Comment:
I wonder if now also need to import scss styles here?
cc @stephenliang 
<img width="443" alt="image" src="https://github.com/user-attachments/assets/96c61f5f-a165-4c19-9987-7ccb7e9ecb38" />